cola/version.py

```diff
@@ -16,9 +20,6 @@
     srcdir = os.path.dirname(os.path.dirname(__file__))
     sys.path.insert(1, srcdir)
 
-from .git import STDOUT  # noqa
-from .decorators import memoize  # noqa
-from ._version import VERSION  # noqa
```

Comment:
Makes sense. A while back in the early python2 days we deferred these imports and it allowed us to run `python cola/version.py`as a standalone script from the source tree.

We also had additional code that printed out the version (used by some of the build stuff at the time) but we got rid of that a while ago but had forgotten about the `__name__ == '__main__'` snippet here.

TODO note for myself: remove the `__main__` block here. This commit is good.. just a reminder to myself for later.

Reply:
Thanks Kurt, adding dependabot for sounds great.